### PR TITLE
Simplified access of getSimpleName

### PR DESCRIPTION
**Pull request by guptasourabh**
instead of calling getClass.getSimpleName() that will check too many conditions inside , we can make it as final String and use it directly.

### CODE REVIEW

File: gson/src/main/java/com/google/gson/DefaultDateTypeAdapter.java [hunk 1]
```diff
@@ -40,6 +40,7 @@
 
   private final DateFormat enUsFormat;
   private final DateFormat localFormat;
+  private final String SIMPLE_NAME = "DefaultDateTypeAdapter";
```
Comment (JakeWharton) [inline gson/src/main/java/com/google/gson/DefaultDateTypeAdapter.java]: static

**Comment by guptasourabh** [comment]
@inder123 for review

File: gson/src/main/java/com/google/gson/DefaultDateTypeAdapter.java [hunk 2]
```diff
@@ -40,6 +40,7 @@
 
   private final DateFormat enUsFormat;
   private final DateFormat localFormat;
+  private final static String SIMPLE_NAME = "DefaultDateTypeAdapter";
```
Comment (JakeWharton) [inline gson/src/main/java/com/google/gson/DefaultDateTypeAdapter.java]: Static fields go at the top above instance fields.

**Comment by guptasourabh** [comment]
@JakeWharton requested changes done

**Comment by inder123** [comment]
👍

File: gson/src/main/java/com/google/gson/DefaultDateTypeAdapter.java [hunk 3]
```diff
@@ -111,7 +113,7 @@ private Date deserializeToDate(JsonElement json) {
   @Override
   public String toString() {
```
Comment (inder123) [inline gson/src/main/java/com/google/gson/DefaultDateTypeAdapter.java]: toString() is mainly used for debugging.

**Pull request merged**